src/mgmt/tests/mocks.c

```diff
@@ -469,3 +469,9 @@ ds_mgmt_dev_set_faulty(uuid_t uuid, Mgmt__DevStateResp *resp)
 {
 	return 0;
 }
+
+void
+ds_mgmt_kill_rank(bool force)
+{
+	return;
+}
```

Comment:
(style)  void function return statements are not generally useful

Reply:
Should probably resolve this. Just making the body empty is perfectly fine. If you want to show that it was called in your tests, you could add some variables like call count for it and increment when it's called.